### ISSUE TITLE
Have AscendBot Automatically Import Issues to GitHub

### ISSUE THREAD

**Issue by VolticFroogo** [mc-ascendancy/ascendancy#13]
Not from the ideas channel, but from the development channel.

Possibly make ideas automatically create issues on GitHub to allow immediate discussion of ideas here, and reduce the workload of admins. I am not sure if utilising the GitHub API would be too easy from the current setup, so it may be practically not worth the invested time.

**Comment by brussels-sprout** [comment]
I don't think the GitHub API will be an issue. The only concern is how to get good titles for each idea. We could have a command that takes the message ID of the idea in #ideas and a title for it.

**Comment by VolticFroogo** [comment]
@brussels-sprout, we'd just have to force the users to format it properly, 50 char max title, then they put two newlines before the description.

Though this relies on everyone actually reading how it works, and I think we know how that can go sometimes.

**Comment by brussels-sprout** [comment]
> Not from the ideas channel, but from the development channel.

Can you explain what you mean here?

**Comment by VolticFroogo** [comment]
@brussels-sprout, oh this is just because at the top it usually says who posted it and from the ideas channel.

But this was my idea, and from the development channel. It has nothing to do with the implementation.

**Comment by brussels-sprout** [comment]
> I don't think the GitHub API will be an issue. The only concern is how to get good titles for each idea. We could have a command that takes the message ID of the idea in #ideas and a title for it.

Alright. What I meant here is we could have users send their ideas in #ideas as they do now. To create the GitHub issue, the mods would only have to run a command that takes the message ID of the idea and the title of the idea (decided by mods). This would save some time and would lead to more organised issues and make it easier for users to sumbit ideas than if users had to write a good title and use the correct formatting.

**Comment by VolticFroogo** [comment]
We could do it that way, but I feel like it would be more effort for mods and less gratifying for the user. Because we could make it immediately post the link to the issue. Besides, if the formatting is wrong, we can manually correct it ourselves.

**Comment by brussels-sprout** [comment]
Yeah that's true. In that case, @connor7000 would need to give mods permissions to delete issues since there will be some spam.

**Comment by brussels-sprout** [comment]
Any thoughts on this idea @connor7000?
